addons/mail/tests/test_link_preview.py

```diff
@@ -88,3 +93,14 @@ def test_02_link_preview_create(self):
                     } for link_preview in message.link_preview_ids]
                 }]
             )
+
+    def test_03_link_preview_create(self):
```

Comment:
? Just to have a somewhat unique/descriptive name
```suggestion
    def test_03_link_preview_create_no_content_type(self):
```